docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - sh
       - -c
       - yarn dev --host 0.0.0.0
-    ports: [ "3000:3000", "24678:24678" ]
+    ports: [ "4321:4321", "24678:24678" ]
```

Comment:
[Astro v3.0 has changed the default port](https://docs.astro.build/en/guides/upgrade-to/v3/#changed-default-port-3000)